T1/Program.cs

```diff
@@ -0,0 +1,98 @@
+﻿namespace Homework_Task1_1
+{
+	internal class Program
+	{
+		static void Main(string[] args)
+		{
+			Console.WriteLine("Enter two integers a and b:");
+			int a = int.Parse(Console.ReadLine());
+			int b = int.Parse(Console.ReadLine());
+
+			Console.WriteLine($"Integers in the range from {a} to {b} with exactly two symbols A in their duodecimal representation:");
+
```

Comment:
Just determine max(a,b) and min (a,b), and use this values in one loop, without if and else